views/index.html

```diff
@@ -1,35 +1,46 @@
 <!DOCTYPE html>
 <html>
+  <head>
+    <title>Exercise Tracker | Free Code Camp</title>
```

Comment:
Sorry, noticed one minor thing:

```suggestion
    <title>Exercise Tracker | freeCodeCamp</title>
```

Not a huge deal, though.